bees/telegrambee/telegrambee.go

```diff
@@ -116,6 +117,11 @@ func (mod *TelegramBee) Run(eventChan chan bees.Event) {
 						Type:  "string",
 						Value: strconv.Itoa(update.Message.From.ID),
 					},
+					{
+						Name:  "timestamp",
+						Type:  "string",
```

Comment:
If we use unix timestamps here, we should treat the parameter as an integer as well. That being said, I think we actually wanna support handling timestamp as time.Time internally and let the conversion routine in placeholders.go do its magic (essentially make that support time.Time and let it convert from/to strings and integers).